examples/resources/awscc_wafv2_regex_pattern_set/wafv2_regex_pattern_set.tf

```
@@ -0,0 +1,17 @@
+resource "awscc_wafv2_regex_pattern_set" "example" {
+
+  regular_expression_list = [
+    "one",
+    "two"
```

```suggestion
    "I[a@]mAB[a@]dRequest",
    "^foobar$"
```

Let's give practical examples

---

Done.